src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__day calendar__day--1"></div>
```

Comment:
There shouldn't be modifiers for `calendar__day`. You should implement modificators for `calendar`:
![image](https://github.com/mate-academy/layout_calendar/assets/86923089/76df8898-ed4b-4cf3-8375-5c5c6b195005)